app/boards/shields/waterfowl/waterfowl.keymap

```diff
@@ -0,0 +1,112 @@
+	&kp N1      &lt 3 DEL  &lt 1 SPACE &kp TAB     &kp N2      		&kp N3  &kp ESC      &kp BSPC    &lt 2 RET  &kp N4
+			>;
+
+			sensor-bindings = <&inc_dec_kp PAGE_UP PAGE_DOWN &inc_dec_kp TAB LS(TAB)>;
+		};
+
+		navnum_layer {
+/* NAVNUM
+ *
+ * ,----------------------------------.                      ,----------------------------------.
+ * |      | PgUp |  UP  | PgDn |      |                      |   /  |   7  |   8  |   9  |   -  |
+ * |------+------+------+------+------|                      |------+------+------+------+------|
+ * | Home | Left | Down | Right| End  |                      |   =  |   4  |   5  |   6  |   +  |
+ * |------+------+------+------+------|  ,-----.    ,-----.  |------+------+------+------+------|
+ * |      |      |  INS |      |      |  |  2  |    |  3  |  |   0  |   1  |   2  |   3  |   *  |
+ * `----------------------------------'  `-----'    `-----'  `----------------------------------'
+ *          ,-----.   ,--------------------.            ,--------------------.   ,-----. 
+ *          |  1  |   | DEL | SPACE | MO(3)|            |  ESC  | BS | ENTER |   |  4  |
+ *          `-----'   `--------------------'            `--------------------'   `-----'
+ */
+			bindings = <
+	&trans   &kp PG_UP &kp UP   &kp PG_DN &trans                &kp FSLH  &kp N7   &kp N8    &kp N9     &kp MINUS
+	&kp HOME &kp LEFT  &kp DOWN &kp RIGHT &kp END               &kp EQUAL &kp N4   &kp N5    &kp N6     &kp PLUS
+	&trans   &trans    &kp INS  &trans    &trans     			&kp N0    &kp N1   &kp N2    &kp N3     &kp ASTERISK
+	&kp N1 &lt 3 DEL &lt 1 SPACE &kp TAB &kp N2      			    &kp N3    &kp ESC  &kp BSPC  &lt 2 RET  &kp N4
+			>;
+
+			sensor-bindings = <&inc_dec_kp PAGE_UP PAGE_DOWN &inc_dec_kp TAB LS(TAB)>;
+		};
+
+		symbol_layer {
+/* SYM
+ *
+ * ,----------------------------------.                      ,----------------------------------.
+ * |   %  |   @  |   [  |   ]  |   \  |                      |   ¦  |   ¬  |   ^  |      |      |
+ * |------+------+------+------+------|                      |------+------+------+------+------|
+ * |   #  |   !  |   (  |   )  |   |  |                      |   _  |   '  |   "  |   ~  |   `  |
+ * |------+------+------+------+------|  ,-----.    ,-----.  |------+------+------+------+------|
+ * |   $  |   £  |   {  |   }  |   &  |  |  2  |    |  3  |  |      |      |      |      |      |
+ * `----------------------------------'  `-----'    `-----'  `----------------------------------'
+ *          ,-----.   ,--------------------.            ,--------------------.   ,-----. 
+ *          |  1  |   | DEL | SPACE | TAB  |            |  ESC  | BS | ENTER |   |  4  |
+ *          `-----'   `--------------------'            `--------------------'   `-----'
+ */
+			bindings = <
+	&kp PRCNT       &kp LS(SQT) &kp LBKT    &kp RBKT &kp NON_US_BSLH        &kp RA(GRAVE)   &kp LS(GRAVE)  &kp CARET   &trans               &trans
```

Comment:
The default keymap seems to be set for a UK layout and this is unusual compared to the other board/shields in the repo. I wanted to point it out but I don't really have an opinion one way or the other on whether this is wrong or not. It will not work as expected with US keyboard layouts.